config/StoredProcedures/GetAllInventory.sql

```diff
@@ -30,4 +31,4 @@ on s1.ProductID = s2.ProductID;
 
 END $$
 
-DELIMITER ;
\ No newline at end of file
+DELIMITER ;
```

Comment:
That looks gross lol. I'll have to really think about that one. It appears to work though.